.multi-thread.yml

```diff
@@ -30,7 +30,7 @@ jobs:
       env:
         container: sgx
         pool: 1es-dcv2-focal
-      cmake_args: "-DCOMPILE_TARGETS=sgx -DWORKER_THREADS=4 -DENABLE_2TX_RECONFIG=ON -DLONG_TESTS=ON"
```

Comment:
Enabling Debug seems good, removing the long tests less so?

Reply:
The SGX job never ran long tests, but I thought they should be enabled, that's why I initially added it. I did hit the 60min job limit though, after which I removed it again... Should I increase the limit?